deploy/roles/python-build/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+# main.yml - Installs Python 2.7 from custom RPM
+---
+- name: Make sure git is installed
+  sudo: yes
+  yum:
+    name: git
+    state: latest
+
+- name: Clone git repo
+  sudo: no
+  git:
+    dest: "~/python27"
+    repo: https://github.com/cfpb/python27-for-el6.git
+
+- name: Run bootstrap script
+  sudo: no
+  command: "sh ./bootstrap.sh"
+  args:
+    chdir: "~/python27"
+
+- name: Build python RPM
```

Comment:
I had imagined rpm building living outside the deployment process entirely. Does it make sense to make a bigger deal of the order things happen here and separate workloads? (Even if we necessarily run this on vagrant up, perhaps it could live elsewhere to more clearly separate it from provisioning code?)